spec/Mapper/Form/LocationMapperSpec.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace spec\EzSystems\HybridPlatformUi\Mapper\Form;
+
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\Repository\Values\Content\Location;
+use PhpSpec\ObjectBehavior;
+
+class LocationMapperSpec extends ObjectBehavior
+{
+    function it_should_map_locations_to_form()
+    {
+        $contentInfo = new ContentInfo(['id' => 1]);
+
+        $locations = [
+            new Location(['id' => 1]),
+            new Location(['id' => 2]),
+        ];
+
+        $data = [
+            'contentId' => 1,
+            'locationIds' => [
+                1 => false,
+                2 => false,
```

Comment:
What do you think to refactoring to variables to make it clear which ones should be the same?